src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/IUnconfiguredProjectTasksService.cs

```diff
@@ -36,5 +69,32 @@ internal interface IUnconfiguredProjectTasksService
         ///     Thrown if the project was already unloaded before this method was invoked.
         /// </exception>
         Task<T> LoadedProjectAsync<T>(Func<Task<T>> action);
+
+        /// <summary>
+        ///     Provides protection for an operation that the project will not be considered loaded in the host before 
+        ///     the completion of some task.
+        /// </summary>
+        /// <typeparam name="T">
+        ///     The type of value returned by the joinable.
+        /// </typeparam>
+        /// <param name="action">
+        ///     The action to execute before the project is considered loaded in the host.
+        /// </param>
+        /// <exception cref="OperationCanceledException">
+        ///     Thrown if the project was already unloaded before this method was invoked.
+        /// </exception>
+        Task<T> PrioritizedLoadedInHostAsync<T>(Func<Task<T>> action);
```

Comment:
One of these is called `PrioritizedLoadedInHostAsync`, and the other `PrioritizedProjectLoadedInHostAsync`?  Aren't they meant to be overloads?

Reply:
Yes they are supposed to be, fixing.